### PR TITLE
Fix the server http://central.maven.org is down error

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .DS_Store
 old
 .idea
+java/images # generated by fish-pepper
```
Comment (JohnNiang) [inline .gitignore:4]: Should we exclude `tomcat/images` as well?

Comment (LinuxSuRen) [reply .gitignore:4]: As you can see [here](https://github.com/kubesphere/s2i-java-container/blob/1ddfcad261c4ed3219b1d0d417cc042cfb7fa558/test.sh#L238). Currently, only the dir `java/images` was generated by fish-pepper.

Comment (JohnNiang) [reply .gitignore:4]: Alright.

**Pull request merged**